crates/services/executor/src/executor.rs

```diff
@@ -676,26 +764,31 @@ where
         T: KeyValueInspect<Column = Column>,
     {
         if execution_data.found_mint {
-            return Err(ExecutorError::MintIsNotLastTransaction)
+            return Err(ExecutorError::MintIsNotLastTransaction);
         }
 
         // Throw a clear error if the transaction id is a duplicate
         if tx_st_transaction
             .storage::<ProcessedTransactions>()
             .contains_key(tx_id)?
         {
-            return Err(ExecutorError::TransactionIdCollision(*tx_id))
+            return Err(ExecutorError::TransactionIdCollision(*tx_id));
         }
 
         let block_height = *header.height();
+        let mut relayed_tx: Option<RelayedTransactionId> = None;
```

Comment:
It looks hacky with combination of the code below=)

<img width="649" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/3a4f867d-0c16-4780-91a1-599514f423ea">